contentcuration/contentcuration/static/js/utils/kVueHelper.js

```diff
@@ -0,0 +1,31 @@
+// assumes kolibriGlobal variable is in window. Attached via Django tags
```

Comment:
KVueHelper here, returns the [mounted vue component](https://vuejs.org/v2/api/#vm-mount).